package.json

```diff
@@ -6,9 +6,12 @@
   "dependencies": {
```

Comment:
Is it a good idea to include this directly in the pod-server repo? Wouldn't it be better to have it as a separate repo that can be passed in so that we keep the core libraries small?

Reply:
We could make it a separate repo but there's admin overhead to that, and even including this Redis code, the pod-server repo is still tiny, so I didn't feel the need for it yet. We can discuss further in https://github.com/inrupt/websockets-pubsub/issues/3 and then always decide to do the split later.

Reply:
Okay that sounds good then